src/pages/home/sidebar/SidebarScreen/FABActionsPopover.js

```diff
@@ -0,0 +1,179 @@
+        this.setState({
+            isCreateMenuActive: false,
+        });
+    }
+
+    render() {
+        // Workspaces are policies with type === 'free'
+        const workspaces = _.filter(this.props.allPolicies, policy => policy && policy.type === CONST.POLICY.TYPE.FREE);
+
+        return (
+            <>
+                <PopoverMenu
+                    onClose={this.hideCreateMenu}
+                    isVisible={this.state.isCreateMenuActive}
+                    anchorPosition={styles.createMenuPositionSidebar}
+                    onItemSelected={this.hideCreateMenu}
+                    fromSidebarMediumScreen={!this.props.isSmallScreenWidth}
+                    menuItems={[
+                        {
+                            icon: Expensicons.ChatBubble,
+                            text: this.props.translate('sidebarScreen.newChat'),
+                            onSelected: () => Navigation.navigate(ROUTES.NEW_CHAT),
+                        },
+                        {
+                            icon: Expensicons.Users,
+                            text: this.props.translate('sidebarScreen.newGroup'),
+                            onSelected: () => Navigation.navigate(ROUTES.NEW_GROUP),
+                        },
+                        ...(Permissions.canUsePolicyRooms(this.props.betas) && workspaces.length ? [
+                            {
+                                icon: Expensicons.Hashtag,
+                                text: this.props.translate('sidebarScreen.newRoom'),
+                                onSelected: () => Navigation.navigate(ROUTES.WORKSPACE_NEW_ROOM),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOUSend(this.props.betas) ? [
+                            {
+                                icon: Expensicons.Send,
+                                text: this.props.translate('iou.sendMoney'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_SEND),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOU(this.props.betas) ? [
+                            {
+                                icon: Expensicons.MoneyCircle,
+                                text: this.props.translate('iou.requestMoney'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_REQUEST),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOU(this.props.betas) ? [
+                            {
+                                icon: Expensicons.Receipt,
+                                text: this.props.translate('iou.splitBill'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_BILL),
+                            },
+                        ] : []),
+                        ...(!Policy.isAdminOfFreePolicy(this.props.allPolicies) ? [
```

Comment:
```suggestion
                        ...(!Policy.hasActiveFreePolicy(this.props.allPolicies) ? [
```
This was updated in recent changes, seems missed out!
https://github.com/Expensify/App/blob/43f611b856856199d4433834a4b146520ab2267b/src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js#L174